Juno-amr.Rmd

```diff
@@ -0,0 +1,123 @@
+
+The output of the pipeline will be collected in the folder ‘output’ within the working directory (the directory where the pipeline is active). If you want the output to be in a different location it is possible to do this with the `--output` parameter.  
+
+**Note** all samples in the directory should be the same species and the same file format.  
+
+### Download the pipeline  
+
+**YOU NEED TO DOWNLOAD THE PIPELINE ONCE OR EVERY TIME YOU WANT TO UPDATE IT**
+
+Please follow the [instructions to download pipelines](#downloading-instructions) from the Juno team of the IDS-bioinformatics group.  
+
+The `r parameters$pipeline_name` pipeline can be found [here](https://github.com/RIVM-bioinformatics/Juno-amr).  
+
+### Start the analysis. Basics
+
+1. Open a terminal. (Applications>terminal).   
+
+2. Enter the folder of the pipeline using:   
+
+```{}
+cd /mnt/scratch_dir/<my_folder>/Juno-amr
+```
+
+3. Run the pipeline  
+```{}
+python3 juno-amr.py -s [species] -i [/mnt/scratch_dir/<my_folder>/fastq_or_fasta_file folder]
+```
+
+Please read the section [What to expect while running a Juno pipeline](#what-to-expect).  
+
+See the section [General Troubleshooting](#general-troubleshooting) for any problems you may encounter.  
+
+**Note:** Do not keep all your data (including results) on the scratch_dir partition. You can add the `--output` parameter to the command in order to place your output somewhere else.  
+
+### Output  
+
+In the output you can find 3 folders:  
+
+* **log:** Log with output and error file from the cluster for each snakemake rule/step that is performed.
+
+* **results_per_sample:** Output produced by ResFinder and PointFinder for each sample. CGE's explanation on the output can be found [here](https://cge.cbs.dtu.dk/services/ResFinder/output.php).
+
+* **summary:** Directory with 4 summary files created from each sample within the results_per_sample folder.
+    * **summary_amr_genes.csv:** Shows the samplename resistance gene, identity, alignment length/gene length and coverage per gene hit.
+
+    * **summary_amr_phenotype.csv:** Shows the type of match for each sample on the resistance for each antimicrobial.
+
+    * **summary_amr_pointfinder_predicition.csv:** - Shows the samplename and a 0(no hit) or 1(hit) prediciton for each mutation.
+
+    * **summary_amr_pointfinder_results** Shows the samplename, mutation, nucleotide change, amino acid change, resistance and PMID for each mutation hit
+
+
+**Note:** If you want your output to be stored in a folder with a different name or location, you can use the option `-o` or `--output` 
+
+```{}
+python3 juno-amr.py -i /mnt/scratch_dir/<my_folder>/<my_data>/ -species salmonella -o /mnt/scratch_dir/<my_folder>/<my_results>/
```

Comment:
I just want to double check that it accepts 'salmonella' as species. I remember having used 'Salmonella_spp' but maybe I am wrong or both are possible or you changed the code. I think the rest looks nice!

Reply:
These are all the databases: 'salmonella', 'plasmodium_falciparum', 'neisseria_gonorrhoeae', 'escherichia_coli', 'klebsiella', 'helicobacter_pylori', 'enterococcus_faecalis', 'mycobacterium_tuberculosis', 'enterococcus_faecium', 'campylobacter', 'staphylococcus_aureus'

I think it is possible you used salmonella_spp, i saw some code that if its a type of salmonella they change it to salmonella, so salmonella_spp = salmonella or something. 

In line 34 here: https://bitbucket.org/genomicepidemiology/resfinder/src/e70ed209bad9e4ae34041a2675a0d454e361fd73/run_resfinder.py?at=master#run_resfinder.py-44:47